packages/features/ee/workflows/pages/index.tsx

```diff
@@ -37,35 +43,197 @@ function WorkflowsPage() {
     },
   });
 
+  const query = trpc.viewer.workflows.getByViewer.useQuery();
+
+  useEffect(() => {
```

Comment:
sets filtered workflows once `checkedFiltereItems` changes

Reply:
Can we fetch this from the server?